### PR TITLE
add missing parameter

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/390

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2411,6 +2412,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableBackToClassic
+
+Enable or disable the link "Return to classic SharePoint", on modern SharePoint list and library pages.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2417]:
```suggestion
Enables or disables the link "Return to classic SharePoint" on modern SharePoint list and library pages.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips copy review , please

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**